src/app/wallet/wallet/send/send-confirmation-modal/send-confirmation-modal.component.html

```diff
@@ -31,7 +31,9 @@
   </div>
 
   <div class="tx-amount">
-    <span class="big">{{ sendAmount.getIntegerPart() }}</span><span class="point">.</span><span class="small">{{ sendAmount.getFractionalPart() }}</span>&ensp;<span class="currency">PART</span>
+    <span class="big">{{ sendAmount.getIntegerPart() }}</span>
+    <span class="point">.</span><span class="small">{{ sendAmount.getFractionalPart() }}</span>&ensp;
```

Comment:
Only show dot if it exists

Amount.ifDotExist()
https://github.com/particl/particl-desktop/blob/de7435504c20660063f6a094114d5e01a38db1ad/src/app/wallet/shared/util/utils.ts#L67-L69